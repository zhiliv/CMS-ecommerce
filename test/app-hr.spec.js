import { shallowMount } from '@vue/test-utils'
import Hr from '@/components/app/hr/hr.vue'

describe('app-hr.js', () => {
  let cmp

  beforeEach(() => {
    cmp = shallowMount(Hr, {
      // Создать поверхностный экземпляр компонента
      propsData: {
        classes: 'simple simple-xl',
        styles: 'background: white',
        id: 'indent'
      },
    })
  })
  it('Содержимое html для app-hr ', () => {
    expect(cmp.element).toMatchSnapshot()
  })
})
