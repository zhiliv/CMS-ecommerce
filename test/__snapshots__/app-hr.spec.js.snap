// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app-hr.js Содержимое html для app-hr  1`] = `
<hr
  class="simple simple-xl"
  id="indent"
  style="background: white;"
/>
`;
